data/expected_cil/arg_call.cil

```diff
@@ -0,0 +1,183 @@
+(sensitivity s0)
+(sensitivityorder (s0))
+(user system_u)
+(role system_r)
+(role object_r)
+(userrole system_u system_r)
+(userrole system_u object_r)
+(userlevel system_u (s0))
+(userrange system_u ((s0) (s0)))
+(handleunknown allow)
+(typeattribute domain)
+(typeattribute resource)
+(type baz)
+(roletype object_r baz)
+(typeattributeset resource (baz))
+(type dom1)
+(roletype system_r dom1)
+(typeattributeset domain (dom1))
+(type dom2)
+(roletype system_r dom2)
+(typeattributeset domain (dom2))
+(type dom3)
+(roletype system_r dom3)
+(typeattributeset domain (dom3))
+(typeattribute foo)
+(typeattributeset resource (foo))
+(type kernel_sid)
+(roletype system_r kernel_sid)
+(typeattributeset domain (kernel_sid))
+(type security_sid)
+(roletype object_r security_sid)
+(typeattributeset resource (security_sid))
+(type unlabeled_sid)
+(roletype object_r unlabeled_sid)
+(typeattributeset resource (unlabeled_sid))
+(type bar1)
+(roletype object_r bar1)
+(typeattributeset foo (bar1))
+(typeattributeset resource (bar1))
+(type bar2)
+(roletype object_r bar2)
+(typeattributeset foo (bar2))
+(typeattributeset resource (bar2))
+(type bar3)
+(roletype object_r bar3)
+(typeattributeset foo (bar3))
+(typeattributeset resource (bar3))
+(macro bar1-read ((type this) (type source)) (allow source this (dir (read))))
+(macro bar2-read ((type this) (type source)) (allow source this (lnk_file (read))))
+(macro bar3-read ((type this) (type source)) (allow source this (chr_file (read))))
+(macro baz-call_source_read ((type this) (type source) (type arg)) 
+;Pushed to callers: (source-read foo arg)
```

Comment:
I really like we leave a bread crumb like this.  Is there any way to put this on the previous line with the macro?  If this is any significant amount of work don't do it; it's not hard enough to parse.

Reply:
It would be pretty straightforward to make it go on the end of the line.  The function would still be split up though because the closing paren is on the next line, and that has to go there so it's not in the comment.

I kind of like putting them on their own line, so that it's easy to see the exceptional bit (the comment)